README.md

```diff
@@ -10,15 +10,11 @@ can join the cluster as a worker node.
 More design details can be explored in the [WMCO enhancement](https://github.com/openshift/enhancements/blob/master/enhancements/windows-containers/windows-machine-config-operator.md).
 
 ## Pre-requisites
-- An OKD/OCP cluster of version 4.6 or newer, running on Azure, AWS or vSphere configured with
-  [hybrid OVN Kubernetes networking](docs/setup-hybrid-OVNKubernetes-cluster.md).
-  - In order to [configure BYOH Windows instances](#configuring-byoh-windows-instances) OKD/OCP 4.8+ and WMCO
-    3.0+ must be used.
 - [Install](https://sdk.operatorframework.io/docs/installation/) operator-sdk v1.6.1
```

Comment:
Is this required for normal usage of the operator? Or just the workflows present in the hacking doc?

Reply:
I moved all this info to the pre-req doc. I thought it's required for normal usage, but tell me if I'm wrong.

Reply:
For regular use the SDK is not needed.